packages/desktop-client/src/components/mobile/MobileNavTabs.tsx

```diff
@@ -20,8 +20,10 @@ import { theme, styles, type CSSProperties } from '../../style';
 import { View } from '../common/View';
 import { useScroll } from '../ScrollProvider';
 
-export const ROW_HEIGHT = 70;
 const COLUMN_COUNT = 3;
+const PILL_HEIGHT = 15;
+export const ROW_HEIGHT = 70;
```

Comment:
We can remove the export here I think if it's no longer imported anywhere. :)

Reply:
There is one more use of `ROW_HEIGHT` in `MobileBudgetTable.jsx`, i will use the new `MOBILE_NAV_HEIGHT` there too.